examples/nvidia_cuda_q/container/Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM 292282985366.dkr.ecr.us-west-2.amazonaws.com/amazon-braket-pytorch-jobs:latest
-RUN python3 -m pip install --upgrade pip
+FROM 292282985366.dkr.ecr.us-west-2.amazonaws.com/amazon-braket-base-jobs:latest
 
-# install cudaq
 ARG SCRIPT_PATH
-ARG CUDAQ_PATH=/opt/conda/lib/python3.10/site-packages
-ENV MPI_PATH=/opt/amazon/openmpi
 
-RUN python3 -m pip install cudaq
-RUN bash "${CUDAQ_PATH}/distributed_interfaces/activate_custom_mpi.sh"
+# install cudaq
+ENV MPI_PATH=/usr/local
+RUN pip install cudaq mpi4py==3.1.6
```

Comment:
```
RUN pip install cudaq mpi4py==3.1.6
```

We can consider moving this to a requirements file for easier local compatibility testing. This let's us:
* let dependabot bump versions eagerly
* locally pull in dependencies from a file instead of copying manually from the dockerfile

Can the `RUN`s be combined to avoid extra layers being created?